copr_builder/copr_project.py

```diff
@@ -48,6 +58,30 @@ def _get_package_version(self, build):
         else:
             raise CoprBuilderError('Cannot extract version from last build. ID: %s' % build.id)
 
+    def _get_project_chroots(self):
+        chroots = self.copr_project.get_project_chroot_list()
+        return set(chroot.name for chroot in chroots)
+
+    def _get_build_chroots(self, build):
+        tasks = build.get_build_tasks()
+        return set(task.chroot_name for task in tasks)
+
+    def _get_chroots_diff(self, project, build):
```

Comment:
I would rename this to something like ``_get_chroots_diff_message`` or something similar. Right now the name seems to give you `set` of diff or `True`/`False` value.

Reply:
Renamed.